### PR TITLE
Downgrade cmake version from 3.15 to 3.5 and Update Readme.md

### PR DESCRIPTION
**Pull request by LokeshBonta**
- Downgraded cmake version from 3.15 to 3.5
- Updated Readme.md

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -193,34 +182,20 @@ err = clGetPlatformIDs(1, &platform_id, NULL);
 
 #### RPP stand-alone code snippet using HIP
 ```c
-<<<<<<< HEAD
-hipMalloc(&d_input, ioBufferSize * sizeof(Rpp8u));
-=======
         hipMalloc(&d_input, ioBufferSize * sizeof(Rpp8u));
->>>>>>> 977ec0a11ae7111a6b8abaa9d219b85733943403
+        hipMalloc(&d_input, ioBufferSize * sizeof(Rpp8u));
```
Comment (rrawther) [inline Readme.md:186]: remove extra line

**Comment by rrawther** [comment]
@LokeshBonta 
We are getting error when building using this PR. Can you please check?

CMake Error at CMakeLists.txt:298 (target_link_libraries):
  Target "modules" of type OBJECT_LIBRARY may not be linked into another
  target.  One may link only to STATIC or SHARED libraries, or to executables
  with the ENABLE_EXPORTS property set.

**Comment by kiritigowda** [comment]
@LokeshBonta can you sync the pull request to TOT to resolve conflicts?

**Comment by kiritigowda** [comment]
#20 includes this PR with additional edits.

**Pull request closed without merging**